java/core/src/java/org/apache/orc/TypeDescription.java

```diff
@@ -664,6 +717,9 @@ public TypeDescription(Category category) {
   private int maxLength = DEFAULT_LENGTH;
   private int precision = DEFAULT_PRECISION;
   private int scale = DEFAULT_SCALE;
+  private String CRS = "OGC:CRS84";
```

Comment:
Since we can change this with `withCRS`, can we define `DEFAULT_CRS` constant variable and use it instead?